apps/multiplatform/common/src/androidMain/kotlin/chat/simplex/common/platform/UI.android.kt

```diff
@@ -80,9 +80,6 @@ actual class GlobalExceptionsHandler: Thread.UncaughtExceptionHandler {
           chatModel.chatId.value = null
           chatModel.chatsContext.chatItems.clearAndNotify()
         }
-        withContext(Dispatchers.Main) {
```

Comment:
this was already present in original code for unclear reason, it seems to simply duplicate the line above